content/pages/contributing.md

```diff
@@ -0,0 +1,432 @@
+> need to install anything, but you will need to set up a free GitHub
+> account. Git is a command line tool that is most likely already
+> installed on your machine, and will need to be run from a “terminal” window, AKA
+> a “shell”.
+
+Using, and even just configuring, Git and GitHub are often the most
+daunting aspects of contributing to a GitHub hosted project. Here
+are the basic steps for Git/GitHub configuration, all of which must
+be performed before the next subsection, forking a repo.
+
+#### GitHub Setup
+
+Create a free GitHub account [here](https://github.com/). Note
+GitHub offers free personal use and paid enterprise accounts. The
+free account is all that is needed.
+
+#### Git Setup
+
+If not already installed on your machine, download and install the
+[latest version of Git](https://git-scm.com/downloads) . Set up
+Git with a user name and your email using the steps below. Note,
+it is advisable that you use the same user name/email as you did
+when setting up your GitHub account, though technically this may
+not be necessary. Once git is installed you will need to open a
+terminal/shell and type the following commands to configure git:
+
+```
+$ git config --global user.name "Your name here"
+$ git config --global user.email "your_email@example.com"
+```
+
+Don’t type the \$. This simply indicates the command line prompt.
+
+#### Configure your environment to authenticate with GitHub from Git
+
+This is a complicated process and there are two authentication
+protocols supported: HTTP or SSH. Either will work fine, but we
+find HTTP to be the easiest to set up. Both processes are described
+in detail on the GitHub site [here](https://docs.github.com/en/github/getting-started-with-github/set-up-git#next-steps-authenticating-with-github-from-git).
+
+For further reading see the [GitHub Getting Started
+Guide](https://docs.github.com/en/github/getting-started-with-github).
+
+### Creating a Python environment that will work with Pythia
+
+Before starting any development, you’ll need to create an isolated
+Python environment that will work for Pythia. When we use the term
+“Python environment” here we are referring to the Python programming
+language plus the myriad packages that go along with it. Because
+there are so many Python packages available, maintaining interoperability
+between them is a huge challenge. To overcome some of these
+difficulties the use of Anaconda or miniconda is required to manage
+your Python ecosystem. These package managers allow you to create
+a separate, custom Python environment for each specific Python set
+of tools. Yes, this unfortunately results in multiple copies of
+Python on your system, but it greatly reduces breaking toolchains
+whenever a change is made to your Python environment (and is more
+reliable than any other solution we’ve encountered). Also, the use
+of Anaconda/Miniconda is standard practice for working with Python
+in general, not simply for using Pythia.
+
+The steps:
```

Comment:
We already have detailed instructions for installing conda and setting up a Python environment in the Foundations book [here](https://foundations.projectpythia.org/foundations/conda.html)

It would be good link to this.

Reply:
Good catch. Thanks!